src/scss/base/variables.scss

```diff
@@ -0,0 +1,15 @@
+$turquiose: #2FBCF7 !default;
+$white: #fff !default;
+$black: #000 !default;
+$error: #EB2A2A !default;
+$focusedBlack: #353535 !default;
+$selectBlack: #1c1c1c !default;
+
+$breakpoints: (
+  'small':  ( max-width:  600px ),
+  'medium': ( max-width:  768px ),
+  'large':  ( max-width: 992px ),
+  'extra-large': ( max-width: 1200px )
```

Comment:
Is there a reason why you chose the 992px and 600px breakpoints?
like, are they common breakpoints?

Reply:
reading about break points i found those value, i don't have a special reason haha